stores/metadata_test.go

```diff
@@ -599,278 +526,6 @@ func TestContractRoots(t *testing.T) {
 	}
 }
 
-// TestRenewContract is a test for AddRenewedContract.
-func TestRenewedContract(t *testing.T) {
```

Comment:
This tests a few regressions we had in the past when renewing contracts. Do we have testing for the same behavior with `AddRenewal`?